mm2src/mm2.rs

```diff
@@ -231,6 +253,9 @@ pub async fn lp_main(params: LpMainParams, ctx_cb: &dyn Fn(u32)) -> Result<(), S
         }
     }
 
+    #[cfg(feature = "custom-swap-locktime")]
+    initialize_payment_locktime(&conf);
```

Comment:
Maybe you should remove this function and just use `If let Some(lt) = conf["payment_locktime"].as_u64()` it's a lot shorter. An independent function is also good for documentation purposes I guess, so I leave this decision to you  :)

Reply:
I'd keep this function 😅